includes/output.inc

```diff
@@ -144,6 +144,11 @@ function drush_format($input, $metadata = NULL, $format = NULL) {
  */
 function dt($string, $args = array()) {
   $output = NULL;
+
+  if (!is_array($args)) {
+    $args = array($args);
```

Comment:
What is this for? $args should never be anything other than an array; are there places where bad values are erroneously passed in? Is this addition related to the other parts of this patch?